packages/core/src/events/DefaultEventHandlers.ts

```diff
@@ -1,4 +1,4 @@
-import { isFunction } from 'lodash';
+import { isFunction } from 'lodash-es';
```

Comment:
Hmm, I believe using the ESM import here would cause issues with our CJS bundle. Maybe we could use https://www.npmjs.com/package/babel-plugin-transform-rename-import to transform `lodash` imports to `lodash-es` for our ESM bundle?

Reply:
Good catch. It looks like react-color did something similar: https://github.com/casesandberg/react-color/pull/749/files. I will look into it.